### PR TITLE
Landing for the company

### PR DESCRIPTION
**Pull request by gluk64**
https://zksync-js-docs--pr36-landing2-urrruov2.web.app/matterlabs.html

### CODE REVIEW

File: docs/.vuepress/components/NavbarML.vue [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<template>
```
Comment (popzxc) [inline docs/.vuepress/components/NavbarML.vue:1]: Don't you think that using a recognizable `vuepress` style for a landing feels "cheap" and lacks individuality?
Also, the default font doesn't look good for a landing, as for me. Even corny Helvetica looks better IMHO.

File: docs/matterlabs.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+<!-- markdownlint-disable -->
+
+<img :src="$withBase('matter_labs_logo_dark.svg')" width="260px"/>
+
+<br>
+<br>
+<br>
+
+## Hey! This is Matter Labs.
```
Comment (popzxc) [inline docs/matterlabs.md:17]: Headers shouldn't have ending periods, should they?

**Review by StanislavBreadless — COMMENTED** [review]
To be frank I do not think that a lot of text is a great landing page.

I know that the old landing page does not mention Zinc and is not informative about our company's new technology or future roadmap. But I think that no company (at least known to me) has a landing page like that because a lot of text with no pictures leaves little (or any) emotional footprint.

Also for a typical user who is used to scrolling the page with their eyes a lot of text is hard to read. Even though it is well-split into paragraphs.

Sorry if I didn't understand the scope of this PR right. But if that is the page for [https://matter-labs.io/](https://matter-labs.io/), then I think it is not good as a permanent solution.

File: docs/matterlabs.md (hunk 2 above)
Comment (StanislavBreadless) [reply docs/matterlabs.md:17]: Although by tradition they should not, I also believe that they are cool and elegant :slightly_smiling_face:

File: docs/matterlabs.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+We are an engineering team passionate about liberty, blockchain, math, and beautiful code. Our head office is
+[in Berlin](https://www.fullnode.berlin/), but we work from all over the world (mostly Europe).
+
+You might know us as humble creators of [zkSync](https://zksync.io).
```
Comment (barakshani) [inline docs/matterlabs.md:22]: As we would say "the creators of" I believe it should be "the humble creators of".

File: docs/matterlabs.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+of the ongoing digital revolution.
+
+zkSync is in production since Summer 2020, with [growing usage](https://zkscan.io). It makes crypto payments cheaper
+today, and it will soon be used to scale arbitrary smart contracts — imagine Solidity in a ZK rollup!
+
+Everything we build is [open source](https://github.com/matter-labs).
+
+### How we work
+
+Matter Labs is a place full of ambitious people: we have tons of freedom but also a very strong results-first mindset —
```
Comment (barakshani) [inline docs/matterlabs.md:39]: I think it is a "result-first mindset".

File: docs/matterlabs.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+what matters is getting things done. We value purposefulness, creativity, directness, minimalism, initiative, and
+BS-intolerance. Being in the business of security, we also pay huge respect to paranoid adversarial thinking.
+
+To take a deeper peek into our values and day-to-day work practices, check out the Matter Labs'
+[Team Handbook](https://www.notion.so/matterlabs/Matter-Labs-Team-Handbook-43342b471fe14f05b2baf250cb7c7a02) and
+[Dev Handbook](https://www.notion.so/matterlabs/Matter-Labs-Dev-Handbook-734f92a38b1d42d3be024886b578ac18).
+
+### Interested to join?
+
+We're always hiring exceptional talent (expect the bar to be high). Check out our
```
Comment (barakshani) [inline docs/matterlabs.md:49]: We're always "looking for" or "open to hire"

**Comment by keriat** [comment]
Can't push. Access denied: https://cln.sh/DvGoqd

I've made a patch, since I can't push anything in this particular repo.

[changes.patch.zip](https://github.com/matter-labs/zksync-docs/files/5859769/changes.patch.zip)

## What was done?

refactor: theme structure updated according to the base documentation of Vuepress.

* header icon now can be customized inside .md
* nav menu now can be customized inside .md
* matterlabs.md now uses home layout
* home layout was slightly refactored to be configurable
* font awesome free icons added in form of SVG-sprite (most light-weight form)
* head menu was changed for the zksync.io and matterlabs accordinly (with the icons)

**Comment by popzxc** [comment]
@keriat push access granted.

**Pull request closed without merging**